### DISCUSSION TITLE
Add a document outline with clickable anchors to the right sidebar of the documentation

### DISCUSSION THREAD

**Discussion by misharigot** [laravel/framework#51142] [Ideas]
Add a document outline with clickable anchors to the right sidebar of the documentation, similar to the 'On this page' section on the Unreal Engine Docs, e.g.: https://dev.epicgames.com/documentation/en-us/unreal-engine/gameplay-classes-in-unreal-engine

This document outline would stay in a fixed position when you scroll down, as opposed to the current table of contents that's present in the current documentation pages, e.g. on https://laravel.com/docs/11.x/validation.

Currently, I have to scroll back up to the table of contents to get an overview of the page, which is not user friendly, especially on large pages such as the one for the validation documentation that I linked above.

**Comment by ndberg** [comment]
I would love to see this as well, as it would make it a lot easier to navigate on the page.
